spec/services/organisations_service_spec.rb

```diff
@@ -0,0 +1,27 @@
+require 'rails_helper'
+
+RSpec.describe OrganisationsService do
+  describe '#find_each' do
+    it 'queries the search API for all organisations' do
+      expected_params = {
+        query: { filter_format: 'organisation' },
+        fields: %w(slug title)
+      }
+      expect(Clients::SearchAPI).to receive(:find_each).with(expected_params)
+
+      subject.find_each {}
+    end
+
+    it 'yields the response' do
+      result = []
+      allow(Clients::SearchAPI).to receive(:find_each).and_yield(:a).and_yield(:b)
+      subject.find_each { |value| result << value }
+
+      expect(result).to eq([:a, :b])
```

Comment:
Addressed in a2d559fac05ee4ff327ef4b732080daf4377dd29 and fb0d8ece361b38e52ca2c5414e246da9af8d1206